src/core/dfn-finder.js

```diff
@@ -0,0 +1,222 @@
+import { pub } from "./pubsubhub";
+import { wrapInner } from "./utils";
+
+const topLevelEntities = new Set([
+  "callback interface",
+  "callback",
+  "dictionary",
+  "enum",
+  "interface mixin",
+  "interface",
+  "typedef",
+]);
+
+// TODO: make these linkable somehow.
+// https://github.com/w3c/respec/issues/999
+// https://github.com/w3c/respec/issues/982
+const unlinkable = new Set(["maplike", "setlike", "stringifier"]);
```

Comment:
Maybe we should trash this. We could link to IDL spec for each of these.

Reply:
@marcoscaceres Wait, do we really want to link to Web IDL spec here? if `maplike` should be linked, why not `attribute`, `getter`, etc.? I think it shouldn't be linked because it's not a type but a syntax.

[Bikeshed does link `stringifier`](https://dom.spec.whatwg.org/#interface-range) but not to the IDL spec.

Can we do this in a separate PR? I think we need to discuss more.